viewer/src/public/migration/types.ts

```diff
@@ -14,23 +14,32 @@ export interface Color {
 
 export interface Cognite3DViewerOptions {
   sdk: CogniteClient;
+
+  /** An existing DOM element that we will render canvas into. */
   domElement?: HTMLElement;
-  noBackground?: boolean;
+
+  /** Send anonymous usage statistics. */
   logMetrics?: boolean;
+
+  /** @deprecated and ignored */
```

Comment:
@j-bjorne I just mentioned how it works right now. Some params are ignored, some will throw. 

Is it ok to deprecate these params from now?

Reply:
What is the effect from a client perspective on marking as deprecated? Will there be a warning on compile or is it strictly a documentation thingy?

Reply:
Documentation + clarification in code like that:

![image](https://user-images.githubusercontent.com/4989157/88261798-4405a400-ccc7-11ea-8953-2ad228f4d5d6.png)

![image](https://user-images.githubusercontent.com/4989157/88261903-731c1580-ccc7-11ea-87e1-e618a3026aa7.png)

Doesn't affect compilation.